Document for Getting started with service binding (#37088)
openshift/openshift-docs: applications/connecting_applications_to_services/getting-started-with-service-binding.adoc

[id="getting-started-with-service-binding"]
= Getting started with service binding
include::modules/common-attributes.adoc[]
include::modules/servicebinding-document-attributes.adoc[]
:context: getting-started-with-service-binding

toc::[]

> {servicebinding-title} manages the data plane for workloads and backing services. This guide provides instructions with examples to help you create a database instance, deploy an application and use {servicebinding-title} to create a binding connection between the application and the database service.

Reviewer: ```suggestion
{servicebinding-title} manages the data plane for workloads and backing services. This guide provides instructions with examples to help you create a database instance, deploy an application, and use {servicebinding-title} to create a binding connection between the application and the database service.
```
Per "Commas" section of IBM Style Guide, "Use commas to separate items in a series of three or more. Use a comma before the conjunction that precedes the final item."

// Prerequisites for getting started with Service Binding Operator
[discrete]
== Prerequisites

* You have access to an {product-title} cluster using an account with `cluster-admin` permissions.

> * You have installed `oc` CLI.

Reviewer: ```suggestion
* You have installed the `oc` CLI.
```
Needs definite article here

* You have installed PostgreSQL `psql` CLI.
* You have installed {servicebinding-title} from OperatorHub.

> * You have installed the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, for example, `my-postgresql` namespace.

Reviewer: ```suggestion
* You have installed the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, such as the `my-postgresql` namespace.
```

//Creating a PostgreSQL database instance
include::modules/sbo-creating-a-postgresql-database-instance.adoc[leveloffset=+1]

//Creating appropriate `ClusterRole` and ClusterRoleBinding` resources

> include::modules/sbo-creating-appropriate-clusterrole-and-clusterrolebinding-resources.adoc[leveloffset=+1]

Reviewer 2: This section is not necessary since crunchy postgres is supported by default via #1043. @pedjak please confirm
